.github/workflows/test-only.yml

```diff
@@ -87,15 +87,66 @@ jobs:
 
       - name: Test changedetection.io container starts+runs basically without error
         run: |
-          docker run -p 5556:5000 -d test-changedetectionio
+          docker run --name test-changedetectionio -p 5556:5000  -d test-changedetectionio
           sleep 3
           # Should return 0 (no error) when grep finds it
           curl -s http://localhost:5556 |grep -q checkbox-uuid
           
           # and IPv6
           curl -s -g -6 "http://[::1]:5556"|grep -q checkbox-uuid
+          docker kill test-changedetectionio
+
+      - name: Test changedetection.io SIGTERM and SIGINT signal shutdown
+        run: |
+          
+          echo SIGINT Shutdown request test
+          docker run --name sig-test -d test-changedetectionio
+          sleep 3
+          echo ">>> Sending SIGINT to sig-test container"
+          docker kill --signal=SIGINT sig-test
+          sleep 3
+          # invert the check (it should be not 0/not running)
+          docker ps
+          docker logs sig-test
+          docker inspect --format '{{json .State.Running}}' sig-test
+          if [ $? -eq 0 ]
```

Comment:
```
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ docker run --name xxxx -d browserless/chrome:latest
cc7740de212848c93969feccd01179b055fadf7543b4b94ce1de6fba8a2a922f
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ docker inspect --format '{{json .State.Running}}' xxxx
true
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ echo $?
0
```

```
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ docker stop xxxx
xxxx
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ docker inspect --format '{{json .State.Running}}' xxxx
false
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ echo $?
0
```

No, this is what I focus.

Reply:
The container is stopped, and false, but status is also 0.

```
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ docker stop xxxx
xxxx
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ docker inspect --format '{{json .State.Running}}' xxxx
false
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ echo $?
0
```

Reply:
This is the log I did my repo. https://github.com/Constantin1489/changedetection.io/actions/runs/6512112609/job/17689125970#step:14:76

If we use $?, this test will always fail.

Reply:
I don't know what is happen to this one.
```
dgtlmoon@dgtlmoon:/var/www/changedetection.io$ docker inspect --format '{{json .State.Running}}' xxxx

Error: No such object: xxxx
```

Reply:
May I suggest the test like this?
```
docker inspect --format '{{json .State.Running}}' sig-test | grep 'false'
# both GNU or BSD grep returns 0 if the line was selected
if [ $? -ne 0 ]
then
  exit 1
fi
```